src/quo2/components/tags/context_tag/view.cljs

```diff
@@ -1,105 +1,206 @@
-       (assoc-in [:style :padding-vertical] 2))
-   [group-avatar/view opts]
-   [text/text
-    {:weight :medium
-     :size   :paragraph-2
-     :style  (:text-style opts)}
-    (str " " label)]])
-
-(defn public-key-tag
-  [params public-key]
-  [base-tag params
-   [text/text
-    {:weight :monospace
-     :size   :paragraph-2}
-    (trim-public-key public-key)]])
-
-(defn context-tag
-  [{:keys [text-style blur? no-avatar-placeholder? text-container-style ellipsize-text?]
-    :as   props}
-   photo
-   name
-   channel-name]
-  (let [text-props        {:weight          :medium
-                           :size            :paragraph-2
-                           :style           (assoc text-style :justify-content :center)
-                           :number-of-lines 1
-                           :ellipsize-mode  :tail}
-        empty-photo?      (nil? photo)
-        avatar-size       :xxs
-        avatar-outer-size (get-in user-avatar-style/sizes [avatar-size :outer])]
-    [base-tag (update-in props [:style :padding-left] #(or % 3))
-     (if (and empty-photo? no-avatar-placeholder?)
-       [rn/view {:style {:width avatar-outer-size}}]
-       [user-avatar/user-avatar
-        {:full-name         name
-         :profile-picture   photo
-         :size              avatar-size
-         :status-indicator? false}])
-     [rn/view {:style (or text-container-style style/context-tag-text-container)}
-      (if ellipsize-text?
-        [rn/view {:style {:flex 1}}
-         [text/text text-props name]]
-        [text/text text-props (str " " name)])
-      (when channel-name
-        [:<>
-         [icons/icon
-          :i/chevron-right
-          {:color (style/context-tag-icon-color blur?)
-           :size  16}]
-         [text/text text-props (str "# " channel-name)]])]]))
-
-(defn user-avatar-tag
-  [params username photo]
-  [context-tag params photo username])
-
-(defn audio-tag
-  [duration params]
-  [base-tag (merge {:style style/audio-tag-container} params)
-   [rn/view {:style style/audio-tag-icon-container}
-    [icons/icon
-     :i/play
-     {:color style/audio-tag-icon-color
-      :size  12}]]
+  (str (subs pk 0 5) "..." (subs pk (- (count pk) 3))))
+
+(defn- address-tag
+  [{:keys [theme size address]}]
+  [rn/view {:style (style/address size)}
    [text/text
-    {:weight :medium
-     :size   :paragraph-2
-     :style  {:margin-left 4
-              :color       (style/audio-tag-text-color (:override-theme params))}}
-    duration]])
-
-(defn community-tag
-  [avatar community-name {:keys [override-theme] :as params}]
-  [context-tag
-   (merge {:style                style/community-tag
-           :text-style           (style/community-tag-text override-theme)
-           :text-container-style style/community-tag-text-container
-           :ellipsize-text?      true}
-          params)
-   avatar
-   community-name])
+    {:style  (style/text theme)
+     :weight :monospace ;; TODO: fix this style
```

Comment:
please create an issue and link the TODO here, if it's an old TODO then just remove it 👍

Reply:
I will create an issue